### PR TITLE
Restart prometheus

### PR DESCRIPTION
**Pull request by anishasthana**
- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [x] For commits that came from upstream, `[UPSTREAM]` has been prepended to the commit message
- [x] JIRA link(s): https://issues.redhat.com/browse/RHODS-1974
- [x] The Jira story is acked
- [x] An entry has been added to the latest build document in [Build Announcements Folder](https://drive.google.com/drive/folders/1sgkK1WZgGo9CXsLizNe0GbAzVKuSKrZL).
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious)
- [x] The developer has manually tested the changes and verified that the changes work.

### CODE REVIEW

**Comment by anishasthana** [comment]
Refer to https://github.com/red-hat-data-services/opendatahub-operator/pull/133 for the latest live builds.

File: deploy.sh [hunk 1]
```diff
@@ -241,21 +241,28 @@ fi
...
 else
   echo "Cluster is not for RHODS engineering or test purposes."
 fi
 
+oc apply -n $ODH_MONITORING_PROJECT -f monitoring/prometheus/prometheus-configs.yaml
+
+prometheus_config=$(oc get cm prometheus -n $ODH_MONITORING_PROJECT -o jsonpath='{.data}' | openssl dgst -binary -sha256 | openssl base64)
+alertmanager_config=$(oc get cm alertmanager -n $ODH_MONITORING_PROJECT -o jsonpath='{.data.alertmanager\.yml}' | openssl dgst -binary -sha256 | openssl base64)
+
+sed -i "s/<prometheus_config_hash>/$prometheus_config/g" monitoring/prometheus/prometheus.yaml
```
Comment (crobby) [inline deploy.sh:260]: When I run this, I get:  sed: -e expression #1, char 29: unknown option to `s'

Any chance this needs to reference prometheus-configs.yaml?

File: deploy.sh [hunk 2]
```diff
@@ -241,21 +241,28 @@ fi
...
+sed -i "s/<alertmanager_config_hash>/$alertmanager_config/g" monitoring/prometheus/prometheus.yaml
```
Comment (crobby) [inline deploy.sh:261]: possible target file name change here too?

**Review by crobby — CHANGES_REQUESTED** [review]
I had a problem when running this...I think it's just targeting the old filenames.  Please take a look

File: deploy.sh (hunk 1 above)
Comment (anishasthana) [reply deploy.sh]: Indeed. I messed up part of the rebase from Lucas's PR :(

**Review by lucferbux — APPROVED** [review]
/lgtm
It works.

**Comment by jgarciao** [comment]
lgtm too (tested livebuild a couple of days ago)

**Pull request merged**